INSTALL.txt

```diff
@@ -52,48 +55,88 @@ Homebrew
 .. code-block:: console
 
     $ brew install graphviz
-    $ pip install --pre pygraphviz
+    $ pip install pygraphviz
 
 Advanced
 ========
 
 The two main difficulties are
-(1) informing pip where Graphviz is installed and
-(2) installing Graphviz.
-
+(1) installing Graphviz and
+(2) informing pip where Graphviz is installed.
 
 Providing path to Graphviz
 --------------------------
 
-You need to know where the binary files, includes files, and library files for Graphviz
-are located on your file system.
-Here are some examples of how you tell pip where these files are.
+If you've installed Graphviz and ``pip`` is unable to find Graphviz, then you need to
+provide ``pip`` with the path where it can find Graphviz.
+To do this, you first need to figure out where the binary files, includes files, and
+library files for Graphviz are located on your file system.
+
+Once you know where you've installed Graphviz, you will need to do something like
+the following.  There is an additional example using Chocolatey on Windows further
+down the page.
 
 MacPorts
 ~~~~~~~~
 
+Note ``port install graphviz-devel`` installs an old developer release of Graphviz.
+Hopefully, the MacPorts packagers will update Graphviz to a recent release.
+Once that happens, you may want to use ``port install graphviz`` instead of
+``port install graphviz-devel`` below.
+There is an open ticket to upgrade MacPorts to version 2.46.0 here:
+https://trac.macports.org/ticket/62165
```

Comment:
```suggestion
.. note:: ``port install graphviz-devel`` installs an old developer release of Graphviz.
Hopefully, the MacPorts packagers will update Graphviz to a recent release.
Once that happens, you may want to use ``port install graphviz`` instead of
``port install graphviz-devel`` below.
There is an open ticket to upgrade MacPorts to version 2.46.0 here:
https://trac.macports.org/ticket/62165
```
Since this is info about an external packager and (hopefully) won't be relevant for too long, it might be nice to put it in some sort of callout.